scripts/genwiki.sh

```diff
@@ -0,0 +1,120 @@
+# Programatically generated Extensions wiki
+# Github @phanirithvij (https://github.com/phanirithvij)
+
+# NOTE: Run this from tachiyomi-extensions/
+# NOTE: E-Hentai has it's image name as laucher.png not launcher.png
+
+# backup wikifile (just in case)
+cp Extensions.md Extensions_backup.md
+# delete current Extensions.md
+echo '' > Extensions.md
+
+TreeURL="https://github.com/inorichi/tachiyomi-extensions/tree/master/src"
+BlobURL="https://github.com/inorichi/tachiyomi-extensions/blob/master"
+
+lang_from_langcode(){
+    case "$1" in
+    "all")
+        echo "All Languages"
+    ;;
+    "en")
+        echo "English"
+    ;;
+    "vi")
+        echo "Vietnamese (Tiếng Việt)"
+    ;;
+    "ru")
+        echo "Russian (Pусский)"
+    ;;
+    "it")
+        echo "Italian (Italiano)"
+    ;;
+    "es")
+        echo "Spanish (Español)"
+    ;;
+    "ko")
+        echo "Korean (한국어)"
+    ;;
+    "ja")
+        echo "Japanese (日本語)"
+    ;;
+    "pt")
+        echo "Portuguese (Português)"
+    ;;
+    "fr")
+        echo "French (Français)"
+    ;;
+    "de")
+        echo "German (Deutsch)"
+    ;;
+    "zh")
+        echo "Chinese (中文)"
+    ;;
+    *)
+        echo "NONE"
+    ;;
+    esac
+}
+
+echo -e "# Extensions\n" >> Extensions.md
+echo -e "All known issues with available sources and extensions\n" >> Extensions.md
+
+echo -e "**Page under construction**" >> Extensions.md
+
+for langcode in $(ls src/)
+do 
+    # Language
+    lang=$(lang_from_langcode $langcode)
+    echo Language $lang
+    echo -e "<details open>\n\t<summary>" >> Extensions.md
+    echo -e "<b>$lang</b>" >> Extensions.md
+    echo -e "\t</summary>\n" >> Extensions.md
```

Comment:
`\n` isn't needed here and only makes an extra new line
```suggestion
    echo -e "\t</summary>" >> Extensions.md
```